staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -233,8 +233,12 @@ var stripSet = fieldpath.NewSet(
 	fieldpath.MakePathOrDie("metadata", "name"),
 	fieldpath.MakePathOrDie("metadata", "namespace"),
 	fieldpath.MakePathOrDie("metadata", "creationTimestamp"),
+	fieldpath.MakePathOrDie("metadata", "deletionTimestamp"),
```

Comment:
I'm wondering if it'd be generally useful to know who/what has set the deletion timestamp.

Reply:
But it's set in the delete handler, which doesn't set the field anyway (maybe we should also fix that?). I don't know.

Reply:
Isn't this a state from which an object cannot recover?
It might be helpful, but I thought there was the audit log feature to better track things like this?
And do we want to change the object pre deletion? Feels a little off to change managed fields on deletion as it means the user deletes a state and then we modify it before really deleting. Wouldn't expect this.

On the other hand it might be a question of consistency to add this to the delete handler.